src/scss/objects/_buttons.scss

```diff
@@ -72,6 +72,8 @@ $btn-rounded-width                  : 38px;
     $btn-primary-bgColor--focus           : $green--darkest;
 
     $btn-secondary-textColor              : $green;
+    $btn-secondary-textColor--active      : $green;
+    $btn-secondary-textColor--hover       : $green;
```

Comment:
Do you need one for `focus` at all?

Reply:
Yep, gets used in the modal 👍

Reply:
ok. Just that the other variables explicitly set a focus style even tho it's the same colour. Also the changelog says added `:hover & :focus`.